roles/kafka/defaults/main.yaml

```diff
@@ -0,0 +1,7 @@
+kafka_namespace: kafka
+
+kafka_install_url: https://strimzi.io/install/latest?namespace=kafka
```

Comment:
```
kafka_install_url: "https://strimzi.io/install/latest?namespace={{ kafka_namespace }}"
```
?

Reply:
sounds good - didn't realise you can use variables anywhere

Reply:
yes, and it is quite useful actually